R/loadPEPdb.R

```diff
@@ -0,0 +1,64 @@
+############################################################################################## 
+# PEP-loading function 
+
+##############################################################################################
+
+
+#' Helper function that allows LOLA to read PEP-formatted databases 
+#' 
+#' @param configFolder     folder where the config.yaml file is located
+#' @param configName       Name of the config.yaml file 
+#' @param useCache            uses simpleCache to cache and load the results    
+#' @return PEPdB contains database location, samples annotation sheet, .yaml config file 
+#' and the regions Granges list 
+#' @export
+#' @examples 
+#' ConfigPath = system.file("extdata", "hg19", "ucsc_examplePEP", "project_config.yaml", package="LOLA")
+#' PEPdb = loadPEPdb_CACHE(configLocation=ConfigPath)
+#' 
+
+
+loadPEPdb = function(configFolder, configName, useCache=TRUE){
```

Comment:
Instead of taking the folder and filename as separate arguments, there should be only 1 argument, which would be the full (relative) path to the config file.